build/release/Makefile

```diff
@@ -121,9 +121,12 @@ promote.helm: $(HELM)
 #	copy existing charts to a temp dir, then combine with new charts, reindex, and upload
 	$(S3_SYNC) s3://$(HELM_S3_BUCKET)/$(HELM_CHANNEL) $(HELM_TEMP)
 	$(S3_SYNC) s3://$(S3_BUCKET)/build/$(BRANCH_NAME)/$(VERSION)/charts $(HELM_TEMP)
```

Comment:
The removing of `*tgz` and renaming the index.yaml needs to come before this line. So move this line 123 to just before the `repo index` on line 128.

Reply:
done